packages/@uppy/core/src/Uppy.js

```diff
@@ -533,6 +540,45 @@ class Uppy {
     }
   }
 
+  #checkAndCreateFileStateObjects (filesToAdd) {
```

Comment:
Only one letter difference is quite hard to read, especially since it's at the end. Thought we were using recursion inside. Since it's not just calling `checkAndCreateFileStateObject` but also other things, we can come up with a new name? Something that would also make the difference clear with `validateRestrictions`, when is that one used and when this?

Reply:
Ok let me think of something